content/ko/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,123 @@
+---
+title: 사이드카 컨테이너
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+사이드카 컨테이너는 주 애플리케이션 컨테이너와 함께 동일한 {{< glossary_tooltip text="파드" term_id="pod" >}} 내에서 실행되는 보조 컨테이너이다.
+이러한 컨테이너들은 로깅, 모니터링, 보안, 또는 데이터 동기화와 같은 
+추가적인 서비스나 기능을 제공함으로써
+주 애플리케이션 컨테이너의 기능을 향상시키거나 확장시키는데 사용된다.
+이는 주 애플리케이션 코드를 직접 변경하지 않고도 가능하다.
+
+<!-- body -->
+
+## 사이드카 컨테이너 활성화
+
+쿠버네티스 1.29부터 기본적으로 활성화된
+`SidecarContainers`라는
+[기능 게이트](/ko/docs/reference/command-line-tools-reference/feature-gates/)를 통해 파드의
+`initContainers` 필드에 나열된 컨테이너에 대한 `restartPolicy`를 지정할 수 있다. 이러한 재시작 가능한 _사이드카_ 컨테이너는
+다른 [초기화 컨테이너](/ko/docs/concepts/workloads/pods/init-containers/) 및 주 애플리케이션 컨테이너와 독립적이다.
+이들은 주 애플리케이션 컨테이너 및
+기타 초기화 컨테이너에 영향을 주지 않고 시작, 중지 또는 재시작될 수 있다.
+
+## 사이드카 컨테이너와 파드 라이프사이클
+
+초기화 컨테이너가 `restartPolicy`를 `Always`로 설정하여 생성되면
+파드의 전체 수명 동안 실행되고 유지된다.
+이는 주 애플리케이션 컨테이너와 분리된 지원 서비스를 실행하는 데 유용하다.
+
+해당 초기화 컨테이너에 `readinessProbe`가 지정된 경우,
+그 결과는 파드의 `ready` 상태를 결정하는 데 사용된다.
+
+이러한 컨테이너들이 초기화 컨테이너로 정의되기 때문에, 다른 초기화 컨테이너와 동일한
+순서 및 순차적 보장의 이점을 누릴 수 있으므로 다른 초기화 컨테이너와
+함께 사용하여 복잡한 파드 초기화 흐름을 만들 수 있다.
+
+일반적인 초기화 컨테이너와 비교할 때, `initContainers` 내에 정의된 사이드카는
+시작된 후에도 계속 실행된다. 이는
+`.spec.initContainers`의 파드 내에 하나 이상의 항목이 있을 때 중요하다. 사이드카 스타일의 초기화 컨테이너가 실행 중일 때 (kubelet이
+해당 초기화 컨테이너의 `started `상태를 true로 설정함), kubelet은 순차적인 `.spec.initContainers` 목록에서
+다음 초기화 컨테이너를 시작한다.
+이 상태는 컨테이너 내에 프로세스가 실행 중이고 시작 프로브가 정의되지 않았거나,
+`startupProbe`가 성공한 결과로 true가 된다.
```

Comment:
[원문]
> That status either becomes true because there is a process running in the
container and no startup probe defined, or as a result of its `startupProbe` succeeding.

원문을 보면

이 상태는
- 컨테이너 내에 프로세스가 실행 중이고 정의된 시작 프로브가 없는 경우 true가 되고,
- 또는, (정의된 시작 프로브가 있는 경우) `startupProbe`의 결과값을 따른다.

같은데.. 어떠신가요.. ㅎㅎ

```suggestion
이 상태는 컨테이너 내에 프로세스가 실행 중이고 정의된 시작 프로브가 없는 경우 true가 되고,
정의된 시작 프로브가 있는 경우 `startupProbe`의 결과값을 따른다.
```